charts/alertmanager/values.yaml

```diff
@@ -185,3 +185,12 @@ configFileLocation: /etc/alertmanager/alertmanager.yml
 
 templates: {}
 #   alertmanager.tmpl: |-
+
+serviceMonitor:
+  enabled: false
```

Comment:
Should this be `true`?

Reply:
doesn't really matter since it's the default